game/src/main/scala/io/aigar/game/Entity.scala

```diff
@@ -4,5 +4,10 @@ import com.github.jpbetz.subspace.Vector2
 
 trait Entity {
   var position = new Vector2(0, 0)
-  var mass = 0
+  var _mass = 0f
```

Comment:
I've put it `protected` since it was not accessible in the child! Thanks for the tip!